src/foam/lib/csv/CSVOutputterImpl.js

```diff
@@ -143,17 +145,24 @@ foam.CLASS({
         { type: 'Context', name: 'x' }
       ],
       code: function(x) {
+        var columnConfig = x.columnConfigToPropertyConverter;
         this.props
-          .map((name) => this.of.getAxiomByName(name))
+          .map((propName) => {
+            var prop = propName;
+            if ( typeof prop === 'string' )
```

Comment:
You can also do: foam.String.isInstance(prop), which is more uniform since a variation of that works for all types which saves you from having to know how to detect each type, which you already know, at least for string.

Reply:
fixed